bundle-sdk/pom.xml

```diff
@@ -82,8 +181,11 @@
                                 <include>software.amazon.awssdk:*</include>
                                 <include>software.amazon:*</include>
                                 <include>software.amazon.s3.accessgrants:*</include>
-                                <inlcude>com.github.ben-manes.caffeine:*</inlcude>
+                                <inlcude>com.github.ben-manes.caffeine:caffeine</inlcude>
+                                <inlcude>com.google.errorprone:error_prone_annotations</inlcude>
+                                <inlcude>org.checkerframework:checker-qual</inlcude>
```

Comment:
`inlcude` -> `include`

Reply:
How come that wasn't caught in testing?

Reply:
I verified manually that the classes did get included. Apparently any string works.